### PR TITLE
Submission Detail Page

### PR DESCRIPTION
**Pull request by yikema**
#### Features

- [x] Basic Info
- [x] Table
- [x] Plot

#### Testing
Tested locally with dummy data

#### Reviewer
@dnc1994

### CODE REVIEW

File: src/components/Cards/ChartCard.vue [hunk 1]
```diff
@@ -6,6 +6,7 @@
           {{title}}
         </slot>
       </h4>
+      <hr>
```
Comment (dnc1994) [inline src/components/Cards/ChartCard.vue:9]: Add a `<hr>` here for consistency with regular cards

File: src/components/Cards/ChartCard.vue [hunk 2]
```diff
@@ -14,7 +15,7 @@
     </template>
     <div>
       <div :id="chartId" class="ct-chart"></div>
-      <div class="footer">
+      <!-- <div class="footer">
```
Comment (dnc1994) [inline src/components/Cards/ChartCard.vue:18]: Remove footer here for consistency with regular cards

File: src/layout/dashboard/DashboardLayout.vue [hunk 3]
```diff
@@ -6,6 +6,7 @@
         <!-- <sidebar-link to="/profile" name="User Profile" icon="ti-user"/> -->
         <sidebar-link to="/history" name="Submission History" icon="ti-view-list-alt"/>
         <sidebar-link to="/submission" name="Submission Detail" icon="ti-info-alt"/>
+        <sidebar-link to="/help" name="Help" icon="ti-help-alt"/>
```
Comment (dnc1994) [inline src/layout/dashboard/DashboardLayout.vue:9]: Changed icon for help page

**Review by dnc1994 — APPROVED** [review]
Approved to merge. Some minor style issues are left for future work.

**Comment by OlegStotsky** [comment]
This looks kinda schizophrenic, lol

**Pull request merged**